portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -28,5 +28,6 @@
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
     response.setContentType("text/html;");
     response.getWriter().println("<h1>Hello world!</h1>");
```

Comment:
Nit: Should we remove hello world?

Reply:
The requirement is not specified.  I think it is ok to just leave it there haha